backend/api/Areas/Keycloak/Profiles/User/AgencyProfile.cs

```diff
@@ -13,6 +12,9 @@ public class AgencyProfile : Profile
         #region Constructors
         public AgencyProfile()
         {
+            CreateMap<Entity.Agency, Entity.Agency>()
```

Comment:
odd.

Reply:
^ See? that's what happens when I have to review 200+ files. After the 20th my attention span drops exponentially LOL

Good catch Devin

Reply:
I agree, but I was testing how we could copy values from one entity to another with AutoMapper.  I think I ended up not doing it.  Probably can remove this.

Reply:
This was required for the Keycloak User management.